packages/frontend/editor-ui/src/composables/useCanvasMapping.ts

```diff
@@ -605,7 +633,8 @@ export function useCanvasMapping({
 						[CanvasConnectionMode.Output]: outputConnections,
 					},
 					issues: {
-						items: nodeIssuesById.value[node.id],
+						execution: nodeIssuesById.value[node.id].execution,
+						validation: nodeIssuesById.value[node.id].validation,
```

Comment:
Could this be like below? So we don't have to calculate `nodeIssuesById`?

```suggestion
						execution: nodeExecutionErrorsById.value[node.id],
						validation: nodeValidationErrorsById.value[node.id],
```